src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -280,17 +287,18 @@ export const selectPortfolioAllocationPercent = createSelector(
 
 export const selectPortfolioTotalFiatBalanceByAccount = createSelector(
   selectPortfolioFiatAccountBalances,
-  accountBalances => {
+  selectBalanceThreshold,
+  (accountBalances, balanceThreshold) => {
     return Object.entries(accountBalances).reduce<{ [k: AccountSpecifier]: string }>(
       (acc, [accountId, balanceObj]) => {
         const totalAccountFiatBalance = Object.values(balanceObj).reduce(
           (totalBalance, currentBalance) => {
-            return bnOrZero(bn(totalBalance).plus(bn(currentBalance))).toFixed(2)
+            return bnOrZero(bn(totalBalance).plus(bn(currentBalance)))
           },
-          '0'
+          bn('0')
```

Comment:
purely out of habit, can you please use `bnOrZero` incase someone changes this in future